acquia_lift.module

```diff
@@ -0,0 +1,262 @@
+      // Set the identity type to the default either as specified by the user or
+      // as the application default.
+      $identity_type = variable_get('acquia_lift_default_identity_type') ?: ACQUIA_LIFT_DEFAULT_IDENTITY_TYPE_DEFAULT;
+      if (!empty($identity_type_param) && isset($all_params[$identity_type_param])) {
+        // If an explicit identity type was passed in the query parameters then
+        //use that instead.
+        $identity_type = check_plain($all_params[$identity_type_param]);
+      }
+      $identity_value = check_plain($all_params[$identity_param]);
+      _acquia_lift_add_meta_tag('identity:' . $identity_type, $identity_value);
+    }
+  }
+
+  $page_context = acquia_lift_get_page_context();
+  foreach ($page_context as $name => $value) {
+    _acquia_lift_add_meta_tag($name, is_array($value) ? implode(',', $value) : $value);
+  }
+
+  // Add the lift js bootstrap loader
+  $js_file = $assets_url . '/lift.js';
+  $attached['js'] = array(
+    'type' => 'file',
+    'file' => $js_file,
+    'weight' => JS_DEFAULT - 10,
+  );
+  $page['page_top']['acquia_lift'] = array(
+    '#attached' => $attached
+  );
+}
+
+/**
+ * Implements hook_node_view().
+ */
+function acquia_lift_node_view($node, $view_mode, $langcode) {
+  acquia_lift_get_page_context($node, $view_mode == 'full');
+}
+
+/**
+ * Implements hook_ctools_render_alter().
+ *
+ * This provides support for panelizer nodes, which don't fire hook_view_alter.
+ */
+function acquia_lift_ctools_render_alter(&$info, &$page, &$context) {
+  $task = $context['task'];
+  if ($page && $task['module'] == 'page_manager' && $task['name'] == 'node_view' && !empty($context['args'])) {
+    $nid = $context['args'][0];
+    if (isset($context['contexts']['argument_entity_id:node_1']) && $context['contexts']['argument_entity_id:node_1']->argument == $nid) {
+      $node = $context['contexts']['argument_entity_id:node_1']->data;
+      acquia_lift_get_page_context($node, TRUE);
+    }
+  }
+}
+
+/**
+ * Gets the page context data.
+ *
+ * @return
+ *   An array of page context data.
+ */
+function acquia_lift_get_page_context($node = NULL, $primary = FALSE) {
+  // Add any page context
+  module_load_include('inc', 'acquia_lift', 'acquia_lift.context');
+  $page_context = _acquia_lift_get_page_context($node, $primary);
+
+  if (!isset($page_context['content_title']) && is_null($node)) {
+    $page_context['content_title'] = drupal_get_title();
+  }
+
+  // Allow other modules to alter the page context to include additional data.
+  drupal_alter('acquia_lift_page_context', $page_context);
+  return $page_context;
+}
+
+/**
+ * Helper function to return a list of udf types.
+ *
+ * @return array
+ *   An array of UDF types defined in Acquia Lift Profile Manager with keys
+ *   as the type name and values as the number allowed.
+ */
+function acquia_lift_get_udf_types() {
+  return array(
+    'person' => 50,
+    'touch' => 20,
+    'event' => 20,
```

Comment:
I talked to Steve. The 21 to 50 UDFs were introduced later. Lift should allow 50 event UDFs, so "50" is what we want here. 

I will update Lift 2.